lib/Runtime/Library/JavascriptPromise.cpp

```diff
@@ -11,11 +11,10 @@ namespace Js
     {
         Assert(type->GetTypeId() == TypeIds_Promise);
 
-        this->status = PromiseStatusCode_Undefined;
-        this->isHandled = false;
+        this->isHandled= false;
```

Comment:
nit: why not initialize these as a part of initialization list of the constructor?

Reply:
fixed.